src/content/guides/entry-advanced.md

```diff
@@ -11,6 +11,20 @@ It is possible to provide different types of files when using an array of values
 
 Let's make an example. We have a PHP application with two page types: home and account. The home page has different layout and non-sharable JavaScript with the rest of the application (account page). We want to output `home.js` and `home.css` from our application files for the home page and `account.js` and `account.css` for account page.
 
+First let's create a directory, initialize npm, [install webpack locally](/guides/installation/#local-installation), and install the tools:
+
+``` bash
+mkdir advanced-entry
+cd advanced-entry
+npm init -y
+npm install --save-dev webpack webpack-cli
```

Comment:
I think we can skip these steps as covered in https://webpack.js.org/guides/installation/#local-installation